packages/@angular/cli/commands/test.ts

```diff
@@ -14,13 +14,15 @@ export interface TestOptions {
   build?: boolean;
   sourcemap?: boolean;
   progress?: boolean;
+  config: string;
 }
 
 
 const TestCommand = EmberTestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
     { name: 'code-coverage', type: Boolean, default: false, aliases: ['cc'] },
+    { name: 'config', type: String, aliases: ['c', 'cf'] },
```

Comment:
Use just the `c` alias instead. I understand the old command had both, but we don't use it so no user would know and this way we don't use up another possible alias.